tests/test_indexer.py

```diff
@@ -268,11 +268,60 @@ def test_subscription_notification_unsuccessful(self):
                          f"WARNING:.*:Failed notification for subscription {subscription_id}"
                          f" for bundle {bundle_id} with transaction id .+ Code: {error_response_code}")
 
-    def test_alias_exists(self):
+    def test_get_index_shape_identifier(self):
+        from dss.events.handlers.index import get_index_shape_identifier
+        index_document = \
+            {
+                'files': {
+                    'assay_json': {
+                        'core': {
+                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/assay.json",
+                            'schema_version': "3.0.0",
+                            'type': "assay"
+                        }
+                    },
+                    'sample_json': {
+                        'core': {
+                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/sample.json",
+                            'schema_version': "3.0.0",
+                            'type': "sample"
+                        }
+                    }
+                }
+            }
```

Comment:
Not a change-request but dict(…) is more concise than `{…}` when the keys are valid identifiers:

```diff
@@ -270,25 +270,18 @@
 
     def test_get_index_shape_identifier(self):
         from dss.events.handlers.index import get_index_shape_identifier
-        index_document = \
-            {
-                'files': {
-                    'assay_json': {
-                        'core': {
-                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/assay.json",
-                            'schema_version': "3.0.0",
-                            'type': "assay"
-                        }
-                    },
-                    'sample_json': {
-                        'core': {
-                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/sample.json",
-                            'schema_version': "3.0.0",
-                            'type': "sample"
-                        }
-                    }
-                }
-            }
+        index_document = dict(
+            files=dict(
+                assay_json=dict(
+                    core=dict(
+                        schema_url="http://hgwdev.soe.ucsc.edu/~kent/hca/schema/assay.json",
+                        schema_version="3.0.0",
+                        type="assay")),
+                sample_json=dict(
+                    core=dict(
+                        schema_url="http://hgwdev.soe.ucsc.edu/~kent/hca/schema/sample.json",
+                        schema_version="3.0.0",
+                        type="sample"))))
         self.assertEqual(get_index_shape_identifier(index_document, logger), "v3")
 
         # Test with mixed/inconsistent metadata schema release versions in the same bundle
```

Also, having the closing curly braces on separate lines is unconventional in the Python world, IIRC.